package.json

```diff
@@ -20,6 +20,8 @@
     "test-chrome": "sh scripts/testcafe-chrome.sh",
     "test-ie": "sh scripts/testcafe-ie.sh",
     "test-jest": "node scripts/test.unit.js --env=jsdom",
+    "test-ui": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe chrome -c 4 ./**/ui.test.js",
+    "test-ui-hl": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' -c 4 ./**/ui.test.js",
```

Comment:
Do we really need new global scripts?

Reply:
Yes, but we will remove some others in other PRs. These here make sense, I used them a lot and always reverted them, but I think the way we develop, those will be the most requested ones during development.